creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
 
 #### Setting the map up
 
-While this can be done on any Minecraft world that a player may have, we will be using a flat world in order to take advantage of using lower values for the coordinates.
+While command blocks can be used in any Minecraft world, we will be using a flat world for a simple and clean environment.
 
 1. Launch Minecraft: Bedrock Edition.
 1. Select **Play**, select **Create New**, then select **Create New World**.
-1. In the world settings, set the world to **Flat World**.
-1. In the world settings, set **Show Coordinates** to **True**.
-1. In the world settings, set **Enabled Cheats** to **True**.
+1. In the world settings, set the world type to **Flat**.
+1. In the world settings, enable **Show Coordinates**.
+1. In the world settings, enable **Activate Cheats**.
 1. Select **Create**.
 
 ### How to get Command Blocks
 
-Since command blocks are not a craft-able block, they will need to be added via the Command Console.
+Command blocks can only be obtained by running a command.
 
-1. Open up the chat dialogue box (T or Enter on Windows 10 OS).
-1. Type the following command: `/give @p command_block`
+1. Open up chat.
+1. Type the following command: `/give @p command_block`.
 
-You will see a single command block that has been added to your inventory.
+You will see a single command block that been added to your inventory.
 
 ## How to use Command Blocks
 
-Command blocks have a wide range of uses within Minecraft, from handling entity spawning locations, rewarding a player for an action, or used to keep score in a multiplayer experience. command blocks also have a lot of customization settings to fit your needs.
-
-When placing a block, you may notice arrows on the side.
+First, place the command block on the ground like any other block. Look at the command block from the side, and you may notice an arrow.
 
 ![Image showcasing CommandBlock Arrows pointing to the right of the image](Media/CommandBlocks/commandblockarrows.png)
```

Comment:
I changed this image because the old one was lit terribly, cropped badly, was in poor resolution, and had a crosshair floating in the middle of it. Also I added glazed terracotta to reinforce the direction of the arrows.

Let me know if the size of the image breaks anything, github doesn't seem to display this image in its markdown preview